cmd/apex/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"net/rpc/jsonrpc"
+)
+
+func main() {
+	// Connect to the UNIX domain socket
+	conn, err := net.Dial("unix", "/var/run/apex.sock")
```

Comment:
This should be `/run/apex.sock` as we found out in bpfd https://github.com/redhat-et/bpfd/commit/ad4c199db395e5a89a1cc243875e18dc1bf9198a

Reply:
Got it, thanks!